### ISSUE TITLE
[1.10.2] Machines non stop looping task % and never finishes

### ISSUE THREAD

**Issue by Gayanfalor** [SonarSonic/Calculator#147]
Hello,

I got this problem with machines never finishing task and % getting restarted all the time.

I dunno how its possible but normaly when i'm alone its working, but when my friend come play with me sometime its working sometimes its looping.

The strange thing is for my friend its working, and when we are both looking into the machine GUI its working for me but as soon as he quits the gui it get back looping.

I dont know if its come from the mod or my client but i'm reporting it here ^^

Here is a small video i made with the  % looping from 0-1% non stop:

https://drive.google.com/file/d/0B9iNaLFrsdUpZGY5VkhMU19QSjg/view?usp=sharing

**Comment by SonarSonic** [comment]
I couldn't recreate this. I've changed what I think might be the issue, when I release the latest version in the next few days let me know how it goes.

**Comment by Gayanfalor** [comment]
Oh ok, i can wait then :p, but i've seen there is got some update to Sonarcore can we update it already?

**Comment by SonarSonic** [comment]
yeah keep using the latest version of everything :)

**Comment by SonarSonic** [comment]
Try the latest version :)

**Comment by Gayanfalor** [comment]
Yep i've just seen the update, we'll put that on server tomorrow thanks

**Comment by zacki1** [comment]
Hi,
i'm experiencing the same issue even with the new patch. i'm playing with a friend in a lan world and noticed this bug is only happening when he is in the world. as soon as he disconnects the progress bar rose as it should.

**Comment by SonarSonic** [comment]
well that's weird

**Comment by zacki1** [comment]
Okay got some new infos. we tried what op already stated and opened the gui of the machine together and the machine worked again. with 3 guys playing, all 3 have to open the gui to get it to work. also the machine keeps working for some time (i guess till someones out of loading range of the machine or something like that)

**Comment by SonarSonic** [comment]
I've attempted another fix, but I still can't recreate this, so I have no idea whats going on.

**Comment by Gayanfalor** [comment]
Ok i'll try it with the new version, i still dont know how it could happens, btw i feel like its something in the client doing this than the server but i can mistake ^^

**Comment by SonarSonic** [comment]
Are the outputs still being made despite the visual glitch?

**Comment by SonarSonic** [comment]
this should have been fixed, is anyone still getting this issue?

**Comment by Gayanfalor** [comment]
I dont know i' have made a new game with updated mods, i have made the furnace for now its working, but i was alone on the serv too ^^.

You want to close the issue ? if i get it back at last i can recreate one ^^
